descqa/readiness_test.py

```diff
@@ -267,8 +270,12 @@ def run_on_single_catalog(self, catalog_instance, catalog_name, output_dir):
                 output_header.append('<span class="fail">{} does not exist!</span>'.format(' or '.join(quantities_needed)))
                 failed_count += 1
                 continue
+            
+            try:
+                data = catalog_instance.get_quantities(quantities_needed)
+            except KeyError:
+                raise ValueError('{} not found'.format(', '.join(quantities_needed)))
```

Comment:
(Lines 273-277) this seems to be a hack that is no longer needed.

Reply:
OK, but since we know that the quantity names will be tuples in the case of composite catalogs why can't we put code in to handle that?

Reply:
For composite catalogs, the readiness test should really only check derived quantities. The native quantities in a composite catalog are only used for accessing underlying catalogs individually, which is not what the readiness test trying to do (imagine that if the add-on fixes a bug in the main catalog, then the readiness test only needs to run on the fixed column, not the original column). Hence, all the tuple quantity names are not actually what the readiness test cares about.